homeassistant/components/bond/utils.py

```diff
@@ -100,10 +102,10 @@ def __init__(self, bond: Bond):
         """Initialize Bond Hub."""
         self.bond: Bond = bond
         self._bridge: Optional[dict] = None
-        self._version: Optional[dict] = None
-        self._devices: Optional[List[BondDevice]] = None
+        self._version: Optional[Dict[str, Any]] = None
```

Comment:
Is it safe to initialize to dict?
```suggestion
        self._version: Dict[str, Any] = {}
```

Reply:
Yes, it should for all of them